tripper/triplestore.py

```diff
@@ -784,16 +506,14 @@ def infer_iri(obj):
     raise TypeError("cannot infer IRI from object {obj!r}")
 
 
-def function_id(func, length=4):
-    """Return a checksum for function `func`.
+def split_iri(iri: str) -> "Tuple[str, str]":
+    """Split iri into namespace and name parts and return them as a tuple."""
```

Comment:
```suggestion
    """Split iri into namespace and name parts and return them as a tuple.

    Parameters:
        iri: The IRI to be split.

    Returns:
        A split IRI. Split into namespace and name.

    """
```

Reply:
I committed this change, but I thought that the Google style docstrings used "Arguments:" instead of "Parameters:". 

However, when I checked it up, it turns up to be "Args:". See
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

They also places "Note:" above the arguments - which seems odd and is the opposite of the NumPy docstrings...

Reply:
Indeed you're right:

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods